src/math/types/seconds_per_liquidity.rs

```diff
@@ -140,7 +140,7 @@ pub mod tests {
                 .unwrap();
             assert_eq!(
                 seconds_per_liquidity.get(),
-                315360000000000000000000000000000000000
+                315360000000000000000000000000000000
```

Comment:
this doesn't look like max value: `log2(315360000000000000000000000000000000) < 2^118`